mathics/builtin/options.py

```diff
@@ -54,8 +54,8 @@ class All(Predefined):
 
     In <url>
     :Plot:
-    /doc/reference-of-built-in-symbols/graphics-and-drawing/plotting-data/plot</url> \
-    </url>, setting the <url>
```

Comment:
Another unrelated fix, but this prevents to have a clean CI.

Reply:
(This also affected #803)

Reply:
Thanks for finding and fixing, but please do this outside of this PR please and just go ahead and get this in.

Reply:
I just fixed it in master.